build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
 
     // Needed for some automated and manual testing (e.g: acceptance tests)
     mavenLocal()
+    maven { url = "https://s01.oss.sonatype.org/content/repositories/comglia-1315" }
```

Comment:
Why do we need it here? Is it a template?

Reply:
To use core sdk from the staging repository, which includes the latest SC V2 changes.

Reply:
Maybe it makes sense to add a TODO here to remove this line before release.